src/main/resources/static/new.css

```diff
@@ -680,20 +680,24 @@ input.toDelegate:checked + span.weekIcon + span.weekIcon {
     border-radius: .3rem;
     padding: .666rem;
     transition: opacity 1s ease-out;
-}
-
-#planned {
     color: green;
     border: 1px solid green;
     background-color: var(--prio2-color);
-    left: 20%;
+    left: 15%;
```

Comment:
this will need fix, ill send you the "new" code

Reply:
I don't understand